persists/backend_server/persist/model.bal

```diff
@@ -0,0 +1,35 @@
+import ballerina/persist as _;
+
+public type Order record {|
+    readonly string orderId;
```

Comment:
Shouldn't we use `id` as the field name here? We have done that for the Cargo entity

Reply:
updated.